Tests/CryptomatorCloudAccessIntegrationTests/README.md

```diff
@@ -33,6 +35,21 @@ If you are building via a CI system, set these secret environment variables acco
 
 ### How to Get the Secrets
 
+#### Box
+
+To get the access token for Box, generate a developer token in the Box Developer Portal. For more detailed instructions, check out the [OAuth 2.0 Documentation from Box](https://developer.box.com/guides/authentication/oauth2/).
```

Comment:
Just wanted to mention here that the developer token shouldn't be mixed up with the access token from OAuth. The access token can actually be extracted at the same time when extracting the refresh token, which you've described below.

However, we should experiment with a solution that doesn't require an access token at all, as I've described in a different comment. My guess is that a refresh token would be enough for Box to get a new access token.

Reply:
Fixed in commit 995defc.